config/model_configs/single_column_precipitation_test.yml

```diff
@@ -0,0 +1,34 @@
+config: "column"
+initial_condition: "PrecipitatingColumn"
+surface_setup: "DefaultExchangeCoefficients"
+z_elem: 200
+z_max: 10000.0
+z_stretch: false
+dt: "10secs"
+t_end: "1500secs"
+dt_save_to_disk: "500secs"
+moist: "nonequil"
+precip_model: "1M"
+precip_upwinding: "first_order"
+vert_diff: "true"
+hyperdiff: "false"
+rayleigh_sponge: true
```

Comment:
I think you may not need vert_diff or rayleigh_sponge here. (It's fine to have them though.)